### DISCUSSION TITLE
API KEY

### DISCUSSION THREAD

**Discussion by appsmartru** [codemagic-ci-cd/codemagic-docs#1209] [Q&A] [answered]
Hello. I can't see the API KEY in the user's control panel. How can this be solved? Thanks!

**Comment by mikhail-tokarev** [comment, marked as answer]
Hi, you can find Codemagic API key in Teams > Personal Account > Integrations

**Comment by appsmartru** [comment]
![image](https://user-images.githubusercontent.com/119721015/205440019-949c37ed-6520-4255-b9b6-239ad92ac73f.png)
The problem is that I don't have such an item

**Reply by mikhail-tokarev** [reply]
you are looking is user preferences instead of `Teams` menu on left navigation bar

**Reply by appsmartru** [reply]
So now you need to enable billing to access the API? There didn't seem to be such obligations before

**Reply by mikhail-tokarev** [reply]
no, you don't need to enable billing to access API key. Can you see `Personal Account` on Teams page?
